bmwam-embedded/src/sensors/DistanceSensor.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <Arduino.h>
+
+#include <sensors.h>
+
+class DistanceSensor {
```

Comment:
So I used a class here...

Would you guys rather do this or just work with namespaces instead? There's a bit of overhead here but I think it looks cleaner